doc-Provisioning_Virtual_Machines_and_Hosts/topics/Catalogs_and_Services.adoc

```diff
@@ -373,178 +373,96 @@ include::common/create-azure-catalog-item.adoc[]
+. Click image:1847.png[Configuration]*Configuration*, then click image:1848.png[Green_Plus_Sign]*Create a new Orchestration Template*.
+. Enter a *Name* and *Description* for your template.
+. Select the template type from the *Template Type* list. The default is Amazon CloudFormation.
+. Select *Draft* to create a draft template.
+. Add your template in the area below for the selected *Template Type*.
+. Click *Add*.
 
-.Create a Service Dialog
+[[editing-orchestration-templates]]
+==== Editing Orchestration Templates
 
-Using a service dialog, you can determine which options the users get to change. When users access a service, the majority of options available to them are preset and cannot be altered. A service dialog exposes some of those options to the user so that even if they are ordering a basic RHEL 7 machine, for example, they can at least choose the amount of memory, virtual CPUs, or other options available to the instance they order. In cases where certain fields must be unique, such as the name of virtual machines in Red Hat Virtualization, users must enter their own unique name for the virtual machine they choose or the operation will fail, so this field must be exposed.
+Complete the following procedure to edit orchestration templates.
 
+. Navigate to menu:Services[Catalogs] and select *Orchestration Templates* in the accordion menu.
+. Select the orchestration template you want to edit from the *All Orchestration Templates* list.
+. Click image:1847.png[Configuration]*Configuration*, then click image:1851.png[Edit_Sign]*Edit this Orchestration Template*.
+. Edit the template as needed.
++
 [NOTE]
 ====
-You cannot create a service without assigning it a service dialog, so even if you do not want to expose any details to users, you must create at least a simple service dialog with a *Submit* and *Cancel* button.
+You can only edit the name and description of a read-only template as there can be stacks associated with the template.
 ====
++
+. Click *Save*.
 
-A service dialog contains three components:
+[[copying-orchestration-templates]]
+==== Copying Orchestration Templates
 
-* One or more *Tabs*.
-* Inside the tabs, one or more *Sections*. 
-* Inside the boxes, one or more *Elements*. Elements are controls that accept input. Elements contain methods, like check boxes, drop-down lists or text fields, to fill in the options on the provisioning dialog.
+Complete the following procedure to copy an orchestration template to create a new template.
 
-[IMPORTANT]
+. Navigate to menu:Services[Catalogs] and select *Orchestration Templates* in the accordion menu.
+. Click image:1847.png[Configuration]*Configuration*, then click image:1859.png[Copy] *Copy this Orchestration Template*.
+. You can make changes to the *Name*, *Description*, and the actual template as required.
++
+[NOTE]
 ====
-The names of the elements must correspond to the options used in the provisioning dialog.
+To create a new template its content must be unique.
```

Comment:
I feel here it might help to expand on what content must be unique. Does changing the name and description make a new template unique or is there more required?

Reply:
Ack, modified the text/note.